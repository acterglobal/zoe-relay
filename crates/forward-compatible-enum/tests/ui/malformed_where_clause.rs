use forward_compatible_enum::ForwardCompatibleEnum;

// Test malformed where clause - should fail to compile
#[derive(ForwardCompatibleEnum)]
#[forward_compatible(serde_serialize = "T: InvalidTrait::")]  // Invalid where clause syntax
pub enum MalformedWhere<T> {
    #[discriminant(0)]
    Value(T),
    
    Unknown { discriminant: u32, data: Vec<u8> },
}

fn main() {}