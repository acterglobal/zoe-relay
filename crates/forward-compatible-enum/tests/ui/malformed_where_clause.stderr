error: unexpected end of input, expected identifier
 --> tests/ui/malformed_where_clause.rs:4:10
  |
4 | #[derive(ForwardCompatibleEnum)]
  |          ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `ForwardCompatibleEnum` (in Nightly builds, run with -Z macro-backtrace for more info)
